flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/backends/hdfs/HDFSBackendImpl.java

```diff
@@ -237,4 +258,95 @@ private HttpResponse doHDFSRequest(String method, String url, ArrayList<Header>
         return response;
     } // doHDFSRequest
     
+    // from here on, consider this link:
+    // http://stackoverflow.com/questions/21629132/httpclient-set-credentials-for-kerberos-authentication
+    private HttpResponse doPrivilegedHDFSRequest(String method, String url, ArrayList<Header> headers,
+            StringEntity entity) throws Exception {
+        try {
+            LoginContext loginContext = new LoginContext("cygnus_krb5_login",
```

Comment:
I tried to put a comment starting by the '`#`' character, but the application complained when running :(